server/log_rpc_server_test.go

```diff
@@ -539,12 +539,12 @@ func TestGetLeavesByLeafValueHashInvalidHash(t *testing.T) {
 
 	// Should have succeeded at RPC level
 	if err != nil {
-		t.Fatalf("GetLeavesByLeafValueHash() = %v", err)
+		t.Fatalf("GetLeavesByLeafValueHash(hash empty) = %v, want nil", err)
```

Comment:
Ah, I wasn't clear enough, let me try again.  `GetLeavesByLeafValueHash()` has two return values, a response and an error code.  In some of the tests you're checking one, in other tests you're checking the other, and it was hard to tell which is which from the output alone.

This was why I suggested `"GetLeavesByLeafValueHash(badHash) = %s, %v; want Api_ERROR, nil"` as an error format string -- it shows both returns.